config/velero/velero.yaml

```diff
@@ -0,0 +1,120 @@
+# This is a placeholder deployment config so operator-sdk add cluster-roles for velero to ClusterServiceVersion
+# It will be removed by yq during make bundle
+kind: Deployment
```

Comment:
My only recommendation for this file is to strip it to make it less something that needs to be maintained.

As I understand it, the only thing that matters is that the deployment's serviceAccountName.

Reply:
@djzager just pulled this from a running development deployment.. nothing much to maintain. Just need a container to be a valid deployment